src/auto-evo/steps/MigrateSpecies.cs

```diff
@@ -0,0 +1,64 @@
+﻿namespace AutoEvo;
+
+using System;
+using System.Linq;
+
+public class MigrateSpecies : IRunStep
```

Comment:
A summary comment on this would be great. If I understood this correctly, then migrations now happen a fixed number of attempts per patch. Is that better than each species be given the same number of migration attempts?

Reply:
I'm not sure better is really objective here, but with the miche data structure it is certainly easier.

Reply:
I guess that is fine then as long as the performance or gameplay implications aren't bad. Still I'd like to see the summary comment added for this class.